omnigibson/utils/constants.py

```diff
@@ -50,17 +50,17 @@ class SemanticClass(IntEnum):
 
 
 # Specific methods for applying / removing particles
-class ParticleModifyMethod(IntEnum):
-    ADJACENCY = 0
-    PROJECTION = 1
+class ParticleModifyMethod(str, Enum):
```

Comment:
whoaaa what is this python magic??

Reply:
You did this magic before and I just copied your code lol.

Reply:
oh LOOL